barretenberg/cpp/src/barretenberg/commitment_schemes/ipa/ipa.hpp

```diff
@@ -807,12 +807,12 @@ template <typename Curve_> class IPA {
         Fr challenge_poly_eval = 1;
         Fr r_pow = r;
         if (uint32_t(log_poly_length.get_value()) > CONST_ECCVM_LOG_N) {
-            throw_or_abort("IPA log_poly_length is too large");
+            throw_or_abort("IPA log_poly_length is too large: " + std::to_string(uint32_t(log_poly_length.get_value())));
```

Comment:
I forgot to update a constant and this was throwing so i figured it's useful for the throw to display the problematic size